### PR TITLE
Update scannermsbuild.properties for 5.10.0

### PR DESCRIPTION
**Pull request by tom-howlett-sonarsource**

### CODE REVIEW

File: scannermsbuild.properties [hunk 1]
```diff
@@ -6,8 +6,8 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarqube.org/latest/analysis/scan/sonarscanner-for-msbuild/
 issueTrackerUrl=https://github.com/SonarSource/sonar-scanner-msbuild/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-msbuild
-archivedVersions=4.7,4.7.1,4.8,4.9,4.10,5.0,5.0.3,5.0.4,5.1,5.2,5.2.1,5.2.2,5.3.1,5.3.2,5.4,5.4.1,5.5.0,5.5.1,5.5.2,5.5.3,5.6.0,5.7.0,5.7.1,5.7.2,5.8.0,5.9.0,5.9.1
-publicVersions=5.9.2
+archivedVersions=4.7,4.7.1,4.8,4.9,4.10,5.0,5.0.3,5.0.4,5.1,5.2,5.2.1,5.2.2,5.3.1,5.3.2,5.4,5.4.1,5.5.0,5.5.1,5.5.2,5.5.3,5.6.0,5.7.0,5.7.1,5.7.2,5.8.0,5.9.0,5.9.1,5.9.2
+publicVersions=5.10.0
```
Comment (christophelevis) [inline scannermsbuild.properties:10]: To keep it simple, we generally don't put the patch digit when it's .0.

File: scannermsbuild.properties [hunk 2]
```diff
@@ -16,6 +16,15 @@ flavors.core20.label=.NET Core 2.1
 flavors.core31.label=.NET Core 3.1
 flavors.framework46.label=.NET Framework 4.6+
 
+5.10.0.description=FIPS Compliance
```
Comment (christophelevis) [inline scannermsbuild.properties:19]: Do we really claim FIPS compliance. Maybe we should have a  more nuanced position: Better FIPS compatibility / compliance?

File: scannermsbuild.properties (hunk 1 above)
Comment (tom-howlett-sonarsource) [reply scannermsbuild.properties]: Updated, thanks

File: scannermsbuild.properties (hunk 2 above)
Comment (tom-howlett-sonarsource) [reply scannermsbuild.properties]: Updated, thanks

**Review by christophelevis — APPROVED** [review]
👍

**Pull request merged**